src/api_server/src/request/machine_configuration.rs

```diff
@@ -15,12 +15,12 @@ pub(crate) fn parse_get_machine_config() -> Result<ParsedRequest, Error> {
 
 pub(crate) fn parse_put_machine_config(body: &Body) -> Result<ParsedRequest, Error> {
     METRICS.put_api_requests.machine_cfg_count.inc();
-    let vm_config = serde_json::from_slice::<VmConfig>(body.raw()).map_err(|err| {
+    let vm_config = serde_json::from_slice::<MachineConfig>(body.raw()).map_err(|err| {
```

Comment:
nitpick: how about renaming variable names as well?
```suggestion
    let machine_config = serde_json::from_slice::<MachineConfig>(body.raw()).map_err(|err| {
```

Reply:
fixed